app/pages/[page].js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
```

Comment:
I moved the logic you had for the form component here. The reason is in nextjs, when you name a file with square brackets like `[page].js` in the _pages_ directory, it sets it up as a dynamic url, so this component will render for /1, /2, etc. This is still a pretty naive implementation and some logic should be added to handle people visiting something like /apple